.github/workflows/cws-btfhub-sync.yml

```diff
@@ -0,0 +1,93 @@
+name: "CWS BTFHub constants sync"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 4 * * 2' # at 4:30 UTC every Tuesday https://crontab.guru/#30_4_*_*_2
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout datadog-agent repository
+        uses: actions/checkout@v3
+        with:
+          ref: main
+
+      - name: Checkout btfhub-archive repository
+        uses: actions/checkout@v3
+        with:
+          repository: aquasecurity/btfhub-archive
+          path: dev/dist/archive
+
+      - name: Install python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.10'
+          cache: 'pip'
+
+      - name: Install go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.17.11'
```

Comment:
Should we wait for https://github.com/DataDog/datadog-agent/pull/13197 to get merged and then use `.go-version` here?

Reply:
Absolutely we can

Reply:
Should be good now